core/src/miner/miner.rs

```diff
@@ -16,7 +16,7 @@
 
 use super::mem_pool::{Error as MemPoolError, MemPool};
 use super::{MinerService, MinerStatus};
-use crate::account_provider::Error as AccountProviderError;
```

Comment:
A typo in the commit message.
minder -> miner

Reply:
Thanks. I fixed it.

Reply:
I fixed it.